syft/rekor/query_test.go

```diff
@@ -0,0 +1,76 @@
+package rekor
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getAndVerifySbomFromUuid(t *testing.T) {
+	tests := []testCase{
```

Comment:
add test where there are no errors?

Reply:
In running the test `one sbom entry, one non-sbom entry` in `Test_CreateRekorSbomRels`, `getAndVerifySbomFromUUID` is called and no errors are expected to be returned... is it OK to test no errors here through a function that calls it?